platform/linux-generic/odp_queue_basic.c

```diff
@@ -104,6 +136,11 @@ static int queue_init_global(void)
 		queue->s.handle = queue_from_index(i);
 	}
 
+	if (read_config_file(queue_glb)) {
+		odp_shm_free(shm);
+		return -1;
```

Comment:
The function uses _odp_libconfig_lookup_int() which reads user's config file first (when present) and otherwise the built in default config (header) file. So, there's always a config file.